fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,24 @@
+ .nav {
+     width: 100vw;
+ }
+
+ .nav .navbar {
+     padding-left: 25px;
+     padding-right: 25px;
+     width: 100%;
+ }
+
+ @media (max-width: 600px) {
+    .nav .navbar {
+        padding-left: 15px;
+        padding-right: 15px;
+        width: 80%;
+    }
+  }
+
+ 
+ .welcome {
+    height: 100vh;
+    background-color:lightgrey;
+    text-align: center;
+}
```

Comment:
missing-end-of-source-newline

Reply:
There's also the need to make sure that your comments are clear and easy to understand. For Example:

This is a clear comment: _The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?_

This is a vague comment: _The README is not descriptive, why don't you make it better?_
Your comment above is not so clear too. Please follow the structure of the clear comment above and reconstruct another comment.